diesel_derives/src/lib.rs

```diff
@@ -393,6 +394,51 @@ pub fn derive_queryable_by_name(input: TokenStream) -> TokenStream {
     expand_proc_macro(input, queryable_by_name::derive)
 }
 
+/// Implements `Selectable`
+///
+/// To implement `Selectable` this derive needs to know the corresponding table
+/// type. By default it uses the `snake_case` type name with an added `s`.
+/// It is possible to change this default by using `#[table_name = "something"]`.
+/// In both cases the module for that table must be in scope.
+/// For example, to derive this for a struct called `User`, you will
+/// likely need a line such as `use schema::users;`
+///
+/// If the name of a field on your struct is different than the column in your
+/// `table!` declaration, or if you are deriving this trait on a tuple struct,
+/// you can annotate the field with `#[column_name = "some_column"]`. For tuple
+/// structs, all fields must have this annotation.
+///
+/// If a field is another struct which implements `Selectable`,
+/// instead of a column, you can annotate that struct with `#[diesel(embed)]`.
+/// Then all fields contained by that inner struct are loaded into
+/// the embedded struct (not necessary the same table).
```

Comment:
```suggestion
/// If a field is another struct which implements `Selectable`,
/// instead of a column, you can annotate that struct with `#[diesel(embed)]`.
/// Then all fields contained by that inner struct are selected as separate tuple. Fields 
/// from a inner struct can come from a different table, as long as the select clause is valid 
/// current query.
```

As `Selectable` does not load anything we should find another formulation here. Maybe just having one or two examples here how the select clause would look like can be helpful?